tailwind.config.js

```diff
@@ -18,6 +19,7 @@ module.exports = {
     },
     borderRadius: {
       default: '6px',
+      sm: '4px',
```

Comment:
🚩 You'll need to check with Peaks whether this can go in - we should be trying to keep aligned with the new GCDS library with the styles in here.

That said, I don't think you will need this style anymore once you use the Pill component (comment above)

Reply:
@p34ks

Reply:
@p34ks and 🐢 chatted about this in standup - we've decided to drop the pill from the modal altogether, just to make less work for now 😄